app/src/renderer/components/common/ActionModal.vue

```diff
@@ -131,7 +131,7 @@ export default {
     selectedSignMethod: `local`,
     signMethods: [
       {
-        key: `(Unsafe) Local Account`,
+        key: `Account Password (Unsafe)`,
```

Comment:
```suggestion
        key: `Local Account Password (Unsafe)`,
```

Reply:
interesting... what is your thinking here? that it's nice for users to know that their account info is local?

currently, our session modals don't say "Local Account" they just say "Account".

Reply:
Yeah, I would like communicate that the account is not stored in a remote location. But it doesn't really matter as this feature will be disabled in production.

Reply:
We can discuss this another time.